README.md

```diff
@@ -26,6 +26,7 @@ Once you have the application installed it should download and alert you that a
 
```

Comment:
Can we add back this section from the current README?

> Use it to:
> 
> view your application state
> show API requests & responses
> perform quick performance benchmarks
> subscribe to parts of your application state
> display messages similar to console.log
> track global errors with source-mapped stack traces including saga stack traces!
> dispatch actions like a government-run mind control experiment
> hot swap your app's state using Redux or mobx-state-tree
> track your sagas
> show image overlay in React Native
> track your Async Storage in React Native
> You plug it into your app as a dev dependency so it adds nothing to your product builds.
> 
> Desktop
> Reactotron on the left, demo React Native app on the right. 

<img width="897" alt="Screenshot 2023-03-14 at 12 07 28 PM" src="https://user-images.githubusercontent.com/37849890/225111331-9d79f5a4-ad39-450c-b349-b2bb7e70824f.png">